kubernetes/deployment-production.tmpl

```diff
@@ -348,3 +348,64 @@ spec:
       port: 80
       targetPort: 80
   type: NodePort
+---
+kind: PersistentVolumeClaim
```

Comment:
I think we need to add a specific reclaim that ensures this disk doesn't get deleted accidentally (upgrading the node etc) policy https://kubernetes.io/docs/concepts/storage/persistent-volumes/#retain

Reply:
My understanding is that the disk would only be deleted if we explicitly delete the persistentvolumeclaim (i.e. actually running `kubectl delete`). Nodes being replaced or us removing the PVC from the deployment template wouldn't delete the PVC.

I'm happy to change the reclaim policy if we think it's necessary, but if we actually want to delete the volume in the future (e.g. if we move the data out of K8) we'd need to remember to manually clean up the data from the volume.